### PR TITLE
fix(twig): normalize twig templates, pt7 - INNO-1019

### PR DESCRIPTION
**Pull request by yhuard**
Updated:
- component-link
- component-link-block
- component-list-item
- component-listing

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 9f54cdf5dccb006e8afb683ca4904a8542b7e280

https://deploy-preview-659--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit c6c60639ce6998d3fa7ef5005a4c512def4fe2d2

https://deploy-preview-659--europa-component-library.netlify.com

File: src/generic/generic-component-list-item/generic-component-list-item.twig [hunk 1]
```diff
@@ -1,96 +1,96 @@
...
-    - "screen_reader" (string): alternative text for screen readers (default: "List item")
-    - "detail" (string): detail concerning the list item (default empty)
-    - "metas" (array): array of meta item, as in 'ecl-meta' component (default empty)
-    - "action" (string): extra actions related to this list item (default empty)
-    - "extra_classes" (string): extra CSS classes to be added
-    - "extra_attributes" (array): extra attributes classes (optional, format: [{ 'name': 'name_of_the_attribute', 'value': 'value_of_the_attribute'}])
+  Parameters:
+    - "variant" (string) (default: ''): can be '', 'date' or 'thumbnail'
+    - "href" (string) (default: '')
+    - "title" (string) (default: ''
```
Comment (kalinchernev) [inline src/generic/generic-component-list-item/generic-component-list-item.twig]: Missing closing bracket.

File: src/systems/ec/ec-component/ec-component-link/ec-component-link.twig [hunk 2]
```diff
@@ -13,28 +21,30 @@
...
 {% set _label = label|default('') %}
+{% set _variant = variant|default('') %}
 
 {# Internal logic - Process properties #}
 
-{% if variant is defined and variant is not empty %}
-  {% if variant is iterable %}
-    {% for var in variant %}
-      {% set _css_class = _css_class ~ ' ' ~ 'ecl-link--' ~ var %}
+{% if _variant is not empty %}
```
Comment (kalinchernev) [inline src/systems/ec/ec-component/ec-component-link/ec-component-link.twig]: Why not making the two checks at once?

**Review by kalinchernev — COMMENTED** [review]
Tiny thing

File: src/systems/ec/ec-component/ec-component-link/ec-component-link.twig (hunk 2 above)
Comment (yhuard) [reply src/systems/ec/ec-component/ec-component-link/ec-component-link.twig]: because of the `else`

**Review by kalinchernev — APPROVED** [review]

**Pull request merged**